frontend/src/services/api.js

```diff
@@ -0,0 +1,110 @@
+import axios from "axios";
+const base_url = import.meta.env.VITE_BASE_URL;
+
+const getToken = () => {
+  const token = localStorage.getItem("token");
+  if (token) {
+    // Proceed with using the token
+    const modifiedToken = token.replace(/"/g, "");
+    return modifiedToken;
+  } else {
+    console.warn("Token is not available. Proceeding without it.");
+    // Handle the absence of the token (e.g., redirect to login)
+  }
+};
+
+export const registerUser = async (user) => {
+  try {
+    const response = await axios.post(`${base_url}/auth/register`, user);
+    return response.data;
+  } catch (error) {
+    return {
+      success: false,
+      message: error.response.data.msg,
+    };
+  }
+};
+
+export const loginUser = async (user) => {
```

Comment:
When I refresh the page, it signs me out. Make sure to persist the user.

Reply:
will do